Assignment-2/src/classes/TrainingSessionManager.cls

```diff
@@ -0,0 +1,171 @@
+     * 
+     * Created by   :   Sajal Mittal
+     * 
+     * Purpose      :   Method to fetch Training_Session__c records
+     * 
+     * param        :   String nameOfSession
+     * 
+     * return       :   void
+    **/
+    public static void readTrainingSession(String nameOfSession) {
+        List<Training_Session__c> listOfSession = new List<Training_Session__c>([SELECT Id, Name, Session_Date__c, Location__c 
+                                                                                            FROM Training_Session__c 
+                                                                                            WHERE Name =: nameOfSession]);
+        System.debug(listOfSession);
+    }
+    /**
+     * Method Name  :   updateTrainingSession
+     * 
+     * Created by   :   Sajal Mittal
+     * 
+     * Purpose      :   Method to perform update DML operation on record of Training_Session__c
+     * 
+     * param        :   Id id, String nameToUpdate
+     * 
+     * return       :   void
+    **/
+    public static void updateTrainingSession(Id id, String nameToUpdate) {
+        List<Training_Session__c> listOfSessionForUpdation = new List<Training_Session__c>([SELECT Id, Name, Session_Date__c, Location__c 
+                                                                                            FROM Training_Session__c 
+                                                                                            WHERE Id =: id]);
+        if(!listOfSessionForUpdation.isEmpty()){
+            for(training_Session__c ts: listOfSessionForUpdation){
+                ts.Name = nameToUpdate;                                                                
+            }
+            if(!listOfSessionForUpdation.isEmpty()){
+                try{
+                    update listOfSessionForUpdation;
+                }catch(DMLException e){
+                    System.debug('Error found: '+e.getMessage());
+                }            
+            }
+        }
+    }
+    /**
+     * Method Name  :   deleteTrainingSession
+     * 
+     * Created by   :   Sajal Mittal
+     * 
+     * Purpose      :   Method to perform Delete DML operation on record of Training_Session__c
+     * 
+     * param        :   Id id
+     * 
+     * return       :   void
+    **/
+    public static void deleteTrainingSession(Id id) {
+        List<Training_Session__c> listOfSessionForDeletion = new List<Training_Session__c>([SELECT Id, Name, Session_Date__c, Location__c 
+                                                                                            FROM Training_Session__c 
+                                                                                            WHERE Id =: id]);
+        if(!listOfSessionForDeletion.isEmpty()){
+            try{
+                delete listOfSessionForDeletion;
+            }catch(DMLException e){
+                System.debug('Error found: '+e.getMessage());
+            }
+        }
+    }
```

Comment:
Give one line space before start (commenting) of the next method. only one line.
Like -> 
                 1. Method Comment
                 2. Method Definition
                 3. One Space line
                 4. Then Second Method Comment
                 ......